templates/workspace_services/mlflow/terraform/web_app.tf

```diff
@@ -0,0 +1,194 @@
+data "azurerm_storage_share" "shared_storage" {
+  name                 = local.shared_storage_share
+  storage_account_name = local.storage_name
+}
+
+resource "azurerm_app_service" "mlflow" {
+  name                   = local.webapp_name
+  location               = data.azurerm_resource_group.ws.location
+  resource_group_name    = data.azurerm_resource_group.ws.name
+  app_service_plan_id    = data.azurerm_app_service_plan.workspace.id
+  https_only             = true
+  vnet_route_all_enabled = true
```

Comment:
not sure why, but this is generating an error in the build validation step in GitHub Actions
you might not need to change anything here, but wanted to call this out

Reply:
I think this needs to be in the site_config block
does this work for you?

Reply:
Put that in the correct section. Will update PR after testing.

Reply:
suggest leave open until PR updated